examples/chainermn/cifar/models/VGG.py

```diff
@@ -53,11 +53,11 @@ class VGG(chainer.Chain):
     no data augmentation.
 
     Args:
-        num_class_labels (int): The number of class labels.
+        n_class_labels (int): The number of class labels.
 
     """
 
-    def __init__(self, num_class_labels=10):
+    def __init__(self, n_class_labels=10):
```

Comment:
I didn't put a procedure to keep the backward compatibility.
Because `VGG` is a module in examples.

Reply:
Technically, you can change argument names while keeping backward compatibility with `parse_kwargs`.
ref: https://github.com/chainer/chainer/pull/5357.

Reply:
Thanks for the suggestion! I added `parse_kwargs` for backward compatibility.
6d1b8d88e